pkg/auth/jwt.go

```diff
@@ -0,0 +1,68 @@
+package auth
+
+import (
+	"fmt"
+	"net/http"
+
+	"NKSS-backend/pkg/handlers"
+
+	_ "github.com/lib/pq"
+	"github.com/golang-jwt/jwt"
+)
+
+func Authenticator(next http.Handler, secret string) http.Handler {
+    
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
why not just return `http.HandlerFunc` like in passDB
https://github.com/NIT-KKR-Student-Support-System/nkss-backend/blob/73e2c9a8b1476ced1c5526bdaee0195d9642b673/pkg/app.go#L47